doc/topics/development/conventions/formulas.rst

```diff
@@ -79,12 +79,12 @@ or zip file of the repository. The directory structure is designed to work with
 
         mkdir -p /srv/formulas
         cd /srv/formulas
-        git clone https://github.com/saltstack-formulas/apache-formula.git
+        git clone https://github.com/saltstack-formulas/apache-formula.git /srv/formulas/apache-formula
```

Comment:
This is not needed at all, because that is exactly what it is going to do after cding into the directory.